# wheel motor command message from DrRobot Robot.
int32 motorcmd1		# left motor encoder command value
int32 motorCmd2	        # right motor encoder Comamnd value
int32 motorCtrl		# motor ctrl method , 0-- PWM, 1-- velocity  2-- position

